### ISSUE TITLE
trace.depMap not mapping to normalized values

### ISSUE THREAD

**Issue by alexisvincent** [ModuleLoader/es-module-loader#530]
It would seem that trace entires depMap values aren't mapping to normalized values.

`SystemJS.loads["https://localhost:3000/app/app.js"].depMap` gives me

```js
{
    ./a.js: "./a.js"
}
```

where I would expect it to give me

```js
{
    ./a.js: "https://localhost:3000/app/a.js"
}
```

Is this expected behaviour?

**Comment by guybedford** [comment]
Thanks, yes this is a bug.

I've added a fix in https://github.com/ModuleLoader/es-module-loader/commit/bde632050af23abe50567010d4bcbb0be40be5df, and included a new SystemJS build at https://github.com/systemjs/systemjs/commit/67a87d1ac2a6c90b3d635804006bf8d909c50c50.

**Comment by guybedford** [comment]
Released in 2.1.1 and SystemJS 0.20.8.
